scripts/create-test-segments.js

```diff
@@ -72,15 +83,7 @@ module.exports = {
       console.log('files in segments dir were changed rebuilding segments data');
       this.build();
     });
-  },
-
-  clean() {
```

Comment:
why was clean removed? Should --clean still be an option above?

Reply:
These are now created in test/dist, which we cleanup

Reply:
So, we don't need lines 37-40?